.github/workflows/pages.yml

```diff
@@ -49,13 +49,15 @@ jobs:
         env:
           JEKYLL_ENV: production
       - name: Upload artifact
+        if: ${{ github.event_name == 'push' && github.repository == 'GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration' && github.ref_name == 'default'}}
         # Automatically uploads an artifact from the './_site' directory by default
         uses: actions/upload-pages-artifact@v1
         with:
           path: "docs/_site/"
 
   # Deployment job
   deploy:
+    if: ${{ github.event_name == 'push' && github.repository == 'GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration' && github.ref_name == 'default'}}
```

Comment:
this will match branches named `default`. We should probably use: `github.event.repository.default_branch` (without `'`)

Reply:
Done